### PR TITLE
doc: doc version update

### PR DESCRIPTION
**Pull request by mia-ko**
Versions updated in conf.py for
- nrf
- mcuboot
- nrfxlib
- tfm
Versions updated in versions.json
Mentions of versions/revision updated
SES links updated to point to v5.68

Manifest file update for nrfxlib TBD

Signed-off-by: Mia Koen <mia.koen@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -354,11 +354,11 @@
 
 .. ### Source: www.segger.com
 
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://www.segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://www.segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x64
-.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://www.segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_macos
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://www.segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://www.segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x64
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://www.segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v568_win_x86.zip
```
Comment (mia-ko) [inline doc/nrf/links.txt:357]: Is v5.68 the correct version to use for SES?

File: doc/versions.json [hunk 2]
```diff
@@ -24,13 +24,13 @@
...
+      "nrfxlib": "1.9.0",
       "zephyr": "2.7.99",
-      "mcuboot": "1.7.99",
-      "nrfx": "2.5",
-      "tfm": "1.3.99",
-      "matter": ""
+      "mcuboot": "1.8.99",
+      "nrfx": "2.7",
+      "tfm": "1.5.0",
+      "matter": "1.9.0"
```
[the remark this answers is not in the document]
Comment (gmarull) [reply doc/versions.json:33]: Matter doesn't have a version afaik.

File: doc/nrf/links.txt [hunk 3]
```diff
@@ -354,11 +354,11 @@
...
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://www.segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v568_win_x64.zip
+.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://www.segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v568_macos_x64.dmg
+.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://www.segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v568_linux_x86.tar.gz
+.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://www.segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v568_linux_x64.tar.gz
```
Comment (gmarull) [inline doc/nrf/links.txt:361]: please split into a separate commit, this is not related to versions update.

**Review by gmarull — CHANGES_REQUESTED** [review]

**Review by gmarull — APPROVED** [review]

File: doc/nrf/links.txt (hunk 1 above)
Comment (umapraseeda) [reply doc/nrf/links.txt:357]: @tejlmand, could you confirm?

File: doc/versions.json [hunk 4]
```diff
@@ -24,12 +24,12 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.8.99",
-      "nrfxlib": "1.8.99",
+      "ncs": "1.9.0",
```
Comment (umapraseeda) [inline doc/versions.json:27]: We should add 1.9.0 above line 4 and below 'latest'.

File: doc/versions.json [hunk 5]
```diff
@@ -24,12 +24,12 @@
...
+      "nrfxlib": "1.9.0",
       "zephyr": "2.7.99",
-      "mcuboot": "1.7.99",
-      "nrfx": "2.5",
-      "tfm": "1.3.99",
+      "mcuboot": "1.8.99",
+      "nrfx": "2.7",
+      "tfm": "1.5.0",
       "matter": ""
     },
```
Comment (umapraseeda) [inline doc/versions.json:34]: We are also missing a section for 1.9.0

File: doc/nrf/links.txt (hunk 1 above)
Comment (tejlmand) [reply doc/nrf/links.txt:357]: that is correct

**Review by tejlmand — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

File: doc/versions.json [hunk 6]
```diff
@@ -1,7 +1,8 @@
 {
   "VERSIONS": [
     "latest",
-    "1.8.0",
+	"1.9.0",
+	"1.8.0",
     "1.7.1",
```
Comment (umapraseeda) [inline doc/versions.json:6]:
```suggestion
    "1.9.0",
    "1.8.0",
    "1.7.1",
```

Comment (mia-ko) [reply doc/versions.json]: Fixed :)

**Review by umapraseeda — APPROVED** [review]

**Comment by shantha-14** [comment]
@mia-ko  The PR needs a rebase and green CI.

**Review by b-gent — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.9.0.rst [hunk 7]
```diff
@@ -7,8 +7,8 @@
    :local:
    :depth: 2
 
-|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF21, nRF52, nRF53, and nRF91 series.
-The SDK includes open source projects (TF-M, MCUboot OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and re-distributed with the SDK.
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, and nRF91 series.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-1.9.0.rst:10]:
```suggestion
|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, and nRF91 Series.
```

Comment (mia-ko) [reply doc/nrf/releases/release-notes-1.9.0.rst]: Fixed

**Review by shantha-14 — APPROVED** [review]
